server/src/dtos/library.dto.ts

```diff
@@ -95,6 +95,9 @@ export class ScanLibraryDto {
 
   @ValidateBoolean({ optional: true })
   refreshAllFiles?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  removeDeleted?: boolean;
```

Comment:
All of these options seems to be mutually exclusive, especially given how they are used in the web. Can we migrate this to an enum instead?

```ts
export enum LibraryAssetScan {
  ADDED='added', // files on disk without assets are added
  REMOVED='removed', // assets without files on disk are removed
  UPDATED='updated', // assets with files that have updated modification times are refreshed
}

Technically refreshing all assets isn't a scan, and I think it could be a separate endpoint. The implementation just needs to queue all assets for the library. You could argue that `libraryId` is an optional property to the "startJob" + metadata extraction job.

If we want to leave it in as another enum under `REFRESH='refresh'` that would be fine with me. 
```